modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectAction/PredefinedValuesTable.tsx

```diff
@@ -198,7 +200,7 @@ export default function PredefinedValuesTable({
 		};
 
 		const deletePredefinedValueField = ({itemData}: {itemData: Item}) => {
-			const [name] = itemData.name.props.children;
+			const name = itemData.name;
```

Comment:
Hey @manuelecastro,you can use [Destructuring assignment](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment) here, we always use it here when it is possible to make the code simpler and more readable, like: `const {name} = itemData;`

Reply:
Hi! Just commited the change you asked :) it really makes the code simpler by doing the destructuring assignment, thanks for the review!